test/browser/browser.sh

```diff
@@ -29,9 +29,14 @@ mkdir -p /root/.ssh
 curl https://raw.githubusercontent.com/cockpit-project/bots/main/machine/identity.pub >> /root/.ssh/authorized_keys
 chmod 600 /root/.ssh/authorized_keys
 
-# HACK: unbreak subuid assignment for new users; see
+# HACK: unbreak subuid assignment for current and new users; see
 # https://bugzilla.redhat.com/show_bug.cgi?id=2382662
-sed -i '/^SUB_.ID_COUNT/ s/ 0/ 65536/' /etc/login.defs
+# https://issues.redhat.com/browse/RHEL-103765
+sed -i '/^SUB_.ID_COUNT/ s/\b0/65536/' /etc/login.defs
+if ! grep -q admin /etc/subuid; then
+    usermod --add-subuids 100000-165535 admin
+    usermod --add-subgids 100000-165535 admin
+fi
```

Comment:
:warning: Don't *ever* do this in production systems. This would have to be dynamic, to avoid uid collision. But there is currently no mechanism or documentation how to *actually* do this in a safe way, so this picks the hack from bugzilla. It's okay for our tests.

Reply:
Update: We don't actually need it in tmt, as we create the `admin` user after fixing login.defs. We only need it for our own bots VM images.